pkg/cmd/server/kubernetes/node/node.go

```diff
@@ -549,7 +327,7 @@ func firstEndpointIPWithNamedPort(endpoints *kapi.Endpoints, port int, portName
 }
 
 // TODO: more generic location
-func includesNamedEndpointPort(ports []kapi.EndpointPort, port int, portName string) bool {
+func includesNamedEndpointPort(ports []kapiv1.EndpointPort, port int, portName string) bool {
```

Comment:
Not wrong, but why bother switching?

Reply:
Allows us to not load an internal client for kubelet (completely).  So the kubelet code path has no internal client logic at all.  Given the desire to split, reduces by one the list of dependencies in NodeConfig (which is net good)